packages/node_modules/@ciscospark/plugin-memberships/src/memberships.js

```diff
@@ -1,8 +1,23 @@
 /*!
- * Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
+ * Copyright (c) 2015-2019 Cisco Systems, Inc. See LICENSE file.
  */
 
 import {SparkPlugin, Page} from '@ciscospark/spark-core';
+import {
+  WEBEX_EVENT,
+  WEBEX_SPACE_TYPE,
+  API_ACTIVITY_FIELD,
+  API_ACTIVITY_VERB,
+  SDK_EVENT_RESOURCE,
+  SDK_EVENT,
+  SDK_EVENT_OWNER,
+  SDK_EVENT_STATUS,
+  hydraTypes,
+  constructHydraId
+} from '@ciscospark/common';
+
+const debug = require('debug')('memberships');
+const _ = require('lodash');
```

Comment:
Instead of importing all of lodash, you can import just what you need, `cloneDeep`. This helps keep the package size down.

Reply:
Will do.  Andrew suggested lodash as opposed to a library free use of JSON.stringify for a deep copy.  I found other spots in the code that import lodash already so thought this approach would not add any package size.

grep -r "lodash" *.js
deps.js:const _ = require('lodash');

Reply:
Got it. If it's already included, that's fine then.